src/opm/io/eclipse/ESmry.cpp

```diff
@@ -58,7 +58,10 @@ ESmry::ESmry(const std::string &filename, bool loadBaseRunData)
     bool formatted=false;
     
     char buff[1000];
-    getcwd( buff, 1000 );
+    
+    if (getcwd( buff, 1000 )==NULL){
```

Comment:
When you use fixed length buffers you could use `PATH_MAX` which is defined in header `limits.h`

Reply:
fixed